ang/civiawards/directives/award.directive.js

```diff
@@ -128,11 +128,13 @@
           return award.case_type_id;
         })
         .catch(function (error) {
-          if (error.error_code === 'already exists') {
-            CRM.alert('Duplicate Award Title. Please choose and different title and try again.', ts('Error'), 'error');
-          } else {
-            CRM.alert(error.error_message, ts('Error'), 'error');
-          }
+          var errorMesssage = error.error_code === 'already exists'
+            ? 'This title is already in use. Please choose another'
+            : error.error_message;
+
+          CRM.alert(ts(errorMesssage), ts('Error'), 'error');
```

Comment:
`ts` can't be used like this. Please refer to this guide: https://docs.civicrm.org/dev/en/latest/translation/#best-practices

Is `error_message` coming from the back end? it should be translated and we should not worry about it. Then the change should be like this:

```js
var errorMesssage = error.error_code === 'already exists'
  ? ts('This title is already in use. Please choose another')
  : error.error_message;

CRM.alert(errorMesssage, ts('Error'), 'error');
```